lib/miq_expression.rb

```diff
@@ -1205,14 +1205,15 @@ def self.reporting_available_fields(model, interval = nil)
       @reporting_available_fields[model.to_s][interval.to_s] ||= VimPerformanceTrend.trend_model_details(interval.to_s)
     elsif model.ends_with?("Performance")
       @reporting_available_fields[model.to_s] ||= {}
-      @reporting_available_fields[model.to_s][interval.to_s] ||= MiqExpression.model_details(model, :include_model => false, :include_tags => true, :interval => interval)
+      @reporting_available_fields[model.to_s][interval.to_s] ||= MiqExpression.model_details(model, :disallow_loading_virtual_custom_attributes => true, :include_model => false, :include_tags => false, :interval => interval)
+      @reporting_available_fields[model.to_s][interval.to_s] + MiqExpression.tag_details(model, model, {}) + _custom_details_for(model, {})
     elsif Chargeback.db_is_chargeback?(model)
       cb_model = Chargeback.report_cb_model(model)
-      @reporting_available_fields[model.to_s] ||=
-        MiqExpression.model_details(model, :include_model => false, :include_tags => true).select { |c| c.last.ends_with?(*ReportController::Reports::Editor::CHARGEBACK_ALLOWED_FIELD_SUFFIXES) } +
-        MiqExpression.tag_details(cb_model, model, {}) + _custom_details_for(cb_model, {})
+      @reporting_available_fields[model.to_s] ||= MiqExpression.model_details(model, :include_model => false, :disallow_loading_virtual_custom_attributes => true, :include_tags => true).select { |c| c.last.ends_with?(*ReportController::Reports::Editor::CHARGEBACK_ALLOWED_FIELD_SUFFIXES) }
```

Comment:
Why is this one still passing `:include_tags => true`? Should it pass `false` and then add the result of 'MiqExpression.tag_details(model, model, {})' in addition to `MiqExpression.tag_details(cb_model, model, {})`?